addons/koji/common/src/main/java/org/commonjava/indy/koji/conf/IndyKojiConfig.java

```diff
@@ -708,6 +714,30 @@ public boolean isEnabledFor( String name )
         return false;
     }
 
+
+    public boolean isEnabledFor( ArtifactStore store )
+    {
+        // MMENG-1262: Check if group enabled the koji pulling in its metadata
+        final String kojiPullVal = store.getMetadata( KOJI_PULL );
+        if ( kojiPullVal != null )
+        {
+            final String pullVal = kojiPullVal.trim();
+            if ( FALSE.equals( parseBoolean( pullVal ) ) || "no".equalsIgnoreCase( pullVal ) )
+            {
+                final Logger logger = LoggerFactory.getLogger( this.getClass() );
+                logger.debug( "Group {} has disabled koji pulling for artifacts with metadata tag.", store.getKey() );
+                return false;
+            }
+            if ( TRUE.equals( parseBoolean( pullVal ) ) || "yes".equalsIgnoreCase( pullVal ) )
+            {
+                return true;
+            }
+        }
+        return isEnabledFor( store.getName() );
```

Comment:
Maybe one more thing here - I would probably rename isEnabledFor to something like isEnabledByNameFor to idicate it is just sub-method of isEnabledFor and not the same with an alternative parameter input (also adding some javadoc could help understand the purpose to people who read this code in a few years.

Reply:
I've switched it to a private method, but sounds good to rename. I'll do it.